turtlesim/src/turtle.cpp

```diff
@@ -148,7 +148,8 @@ bool Turtle::update(double dt, QPainter& path_painter, const QImage& path_image,
 
   QPointF old_pos = pos_;
 
-  orient_ = std::fmod(orient_ + ang_vel_ * dt, 2*PI);
+  orient_ = orient_ + ang_vel_ * dt;
+  orient_ -= 2*PI * std::floor((orient_ + PI)/(2*PI));
```

Comment:
Can you please add a comment above this line briefly describing what the line does for future users reading the code.